roles/eoracle/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# maintains directories and paths
+eoracle_home_dir: /opt/{{ service_identifier }}
```

Comment:
is this change include in this PR https://github.com/encapsulate-xyz/eigenlayer-ansible/pull/3#discussion_r2043540238 ?